src/app/core/common-components/entity-form/entity-form/entity-form.component.scss

```diff
@@ -16,6 +16,9 @@
      of the columns-wrapper class */
   mat-form-field {
     width: 100%;
+    @media screen and (min-width: 1440px) {
+      width: 60%;
+    }
```

Comment:
is there a special reason why you decided to use `@media` rather than a simple `max-width: 864px` (= 60% * 1440)?

Reply:
not a specific reason, but the thought I had was- if we directly use max-width: 864px, and the screen is wider than 1440px (which is rare), there would be a lot of gap between the form fields and the save button due to the fixed width.

That’s why I preferred using a media query with a 60% width to make it responsive for larger screens.